specification/communication/data-plane/MediaComposition/readme.md

```diff
@@ -0,0 +1,66 @@
+# communicationservices
+
+> see https://aka.ms/autorest
+
+This is the AutoRest configuration file for communicationservices.
+
+## Getting Started
+
+To build the SDKs for My API, simply install AutoRest via `npm` (`npm install -g autorest`) and then run:
+
+> `autorest readme.md`
+
+To see additional help and options, run:
+
+> `autorest --help`
+
+For other options on installation see [Installing AutoRest](https://aka.ms/autorest/install) on the AutoRest github page.
+
+---
+
+## Configuration
+
+### Basic Information
+
+These are the global settings for the communicationservices.
+
+```yaml
+openapi-type: data-plane
+tag: package-2021-12-31-preview
+```
+
+### Tag: package-2021-12-31-preview
+
+These settings apply only when `--tag=package-2021-12-31-preview` is specified on the command line.
+
+```yaml $(tag) == 'package-2021-12-31-preview'
+input-file:
+  - preview/2021-12-31-preview/CommunicationMediaComposition.json
+title:
+  Azure Communication Services
+```
+
+---
+
+# Code Generation
+
+## Swagger to SDK
+
+This section describes what SDK should be generated by the automatic system.
+This is not used by Autorest itself.
+
+## Python
+
+See configuration in [readme.python.md](./readme.python.md)
+
+## Ruby
```

Comment:
I dont think Ruby is a tier 1 language. We didn't have any Ruby repos back when working on SDKs.

Reply:
Not sure but I removed it as we have no plans for it as of now.